### PR TITLE
imx7ulp uboot 2021.04 fixes

### PR DESCRIPTION
**Pull request by vanmaegima**
Please do not submit a Pull Request via github.  Our project makes use of
mailing lists for patch submission and review.  For more details please
see https://www.denx.de/wiki/U-Boot/Patches

### CODE REVIEW

**Comment by vanmaegima** [comment]
DO NOT merge before https://github.com/foundriesio/u-boot/pull/110

**Review by MrCry0 — APPROVED** [review]
LGTM

**Review by angolini — APPROVED** [review]

File: include/configs/imx7ulp_spl.h [hunk 1]
```diff
@@ -8,7 +8,7 @@
 #ifndef __IMX7ULP_SPL_CONFIG_H
 #define __IMX7ULP_SPL_CONFIG_H
 
-#if CONFIG_IS_ENABLED(SPL)
+#ifdef CONFIG_SPL
```
Comment (Tim-Anderson) [inline include/configs/imx7ulp_spl.h:11]: Not sure I understand the difference here. Is this to match upstream?

**Review by Tim-Anderson — COMMENTED** [review]
left comments on the #ifdef stuff

File: include/configs/imx7ulp_spl.h (hunk 1 above)
Comment (vanmaegima) [reply include/configs/imx7ulp_spl.h:11]: I'm not sure why but SPL doesn't work with the "if CONFIG_IS_ENABLED" logic. imx7_spl.h uses "ifdef" instead and that fixed my issues.

**Review by Tim-Anderson — APPROVED** [review]
LGTM

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**